packages/jest-haste-map/src/lib/getPlatformExtension.js

```diff
@@ -28,7 +28,8 @@ function getPlatformExtension(
   }
   const platform = file.substring(secondToLast + 1, last);
   // If an overriding platform array is passed, check that first
-  if (platforms && platforms.includes(platform)) {
+
```

Comment:
nitpick: this newline rubs me the wrong way :P

@andrewimm are you following up with turning platforms into a Set in the jest-haste-map constructor? I think we can make it so that the constructor accepts either a set or array and converts an array into a Set. That way it stays backwards compatible.